### ISSUE TITLE
Seil should be able to target overrides to external keyboard only, like Karabiner.

### ISSUE THREAD

**Issue by changemewtf** [tekezo/Seil#37]
I'm trying to remap Caps Lock to Control on a Corsair K70B keyboard. Using the builtin System Preferences -> Keyboard -> Modifier Keys setting doesn't work.

However, since Caps Lock _must_ be remapped using Seil, not Karabiner, this means I don't have access to the "Don't remap an internal keyboard" setting that Karabiner provides.

What I want is to remap the keys on the external keyboard without affecting _any_ of the keys on the builtin keyboard at all. Otherwise, not only will the internal keyboard be incorrectly remapped while the external is plugged in, I'll have to kill and restart Seil every time I unplug it or plug it back in!
